src/consensus/pbft_chain.cpp

```diff
@@ -111,7 +111,7 @@ PbftBlockCert::PbftBlockCert(PbftBlock const& pbft_blk, std::vector<Vote> const&
 
 PbftBlockCert::PbftBlockCert(dev::RLP const& rlp) {
   pbft_blk.reset(new PbftBlock(rlp[0]));
-  for (auto const& el : rlp[1]) {
+  for (auto const el : rlp[1]) {
```

Comment:
are we sure we are not making copies in these rlp loops when not using & ?

Reply:
it was changed because of warnings that become errors
```error: loop variable 'n' is always a copy because the range of type 'dev::RLP' does not return a reference [-Werror,-Wrange-loop-analysis]```